packages/block-library/src/freeform/editor.scss

```diff
@@ -307,11 +307,10 @@ div[data-type="core/freeform"] {
 	// On mobile, toolbars go edge to edge.
 	padding: 0;
 
-	// By using the `data-type` attribute in this selector it ensures these styles only appear
-	// when the block itself is selected, without it they would also show when a parent block
-	// is selected.
-	div[data-type="core/freeform"].is-selected &,
-	div[data-type="core/freeform"].is-typing & {
```

Comment:
This rule doesn't do anything. When `is-typing` was added to a block previously, it would also have the `is-selected`. It isn't possible for a block to be typed in without it also being selected. This seems like a remnant from old block classes when we would omit the `is-selected` class when typing in a block to remove the selected styles.